### ISSUE TITLE
[Talent Bug] Playing the new Hero

### ISSUE THREAD

**Issue by deathpwned** [ValveSoftware/Dota2-Gameplay#22581]
### Description

I was playing Kez, then suddenly midway was choosing my talents, I suddenly clicked techies(while choosing talent tree), my talents turned to Techies(lvl 15 onwards of talent tree), display bug or I don't know. I don't have a screenshot of it, but I tried demo hero, and I got this(even after restarting DotA 2).

### Example Match ID (and possibly Timestamp)

8024538405

### Screenshots

![image](https://github.com/user-attachments/assets/09aa2d96-f911-4a6b-a458-498f41d2ccc7)

**Comment by SlayerGkc** [comment]
the bug happens to "steal" talents from any hero you click on after entering game and the hero you click on should be eligible to skill talents seems like, but you cant actually click on the new talents you get by "stealing", its unresponsive, i thought it might just be a visual bug and would work as intended underneath if i clicked on the wrongly displayed talent but they don't respond to any input not the hotkeys nor the mouse. This is replicable every time in demo mode.

**Comment by nnein12** [comment]
experienced the same thing
https://youtu.be/bmDC66Lz_a8

**Comment by deathpwned** [comment]
> experienced the same thing https://youtu.be/bmDC66Lz_a8

yes thats what happen to me :D i believe it got fixed after my game, there was an update after the game
